src/main/java/com/gmail/nossr50/runnables/skills/RuptureTask.java

```diff
@@ -44,20 +46,32 @@ public void run() {
             //Rupture hasn't ended yet
             if(ruptureTick < expireTick) {
                 //Is it time to damage?
-                if(damageTickTracker >= DAMAGE_TICK_INTERVAL) {
-                    damageTickTracker = 0; //Reset
-                    ParticleEffectUtils.playBleedEffect(targetEntity); //Animate
-
-                    if(targetEntity.getHealth() > 0.01) {
-                        double healthBeforeRuptureIsApplied = targetEntity.getHealth();
-                        double damagedHealth = healthBeforeRuptureIsApplied - calculateAdjustedTickDamage();
-
-                        if(damagedHealth <= 0) {
-                            mcMMO.p.getLogger().severe("DEBUG: Miscalculating Rupture tick damage");
-                        } else {
-                            targetEntity.setHealth(damagedHealth); //Hurt entity without the unwanted side effects of damage()
-                        }
-                    }
+                if(damageTickTracker < DAMAGE_TICK_INTERVAL)
+                    return;
```

Comment:
What's the point of changing the logic to something that achieves the same thing?